modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.js

```diff
@@ -513,6 +513,7 @@ const FrontendDataSet = ({
 					activeDelta={paginationDelta}
 					activePage={pageNumber}
 					deltas={pagination?.deltas}
+					disableEllipsis={items.length / paginationDelta - 5 > 999}
```

Comment:
We can probably drop `-5`, or set the oddly specific `items.length / paginationDelta > 1004` :slightly_smiling_face: